Terminal.Gui/Views/ComboBox.cs

```diff
@@ -122,7 +122,7 @@ private void Initialize()
 					SetValue ((string)searchset [listview.SelectedItem]);
 			};
 
-			Application.Loaded += (Application.ResizedEventArgs a) => {
+			Loaded += (Application.ResizedEventArgs a) => {
```

Comment:
This will not get called in the 'AllViewsTest` scenario. We should not encourage people to do this.

Reply:
I know he will not be called, I had said this before in another comment (https://github.com/migueldeicaza/gui.cs/pull/710#issuecomment-648150447). I will present another alternative that will call a view as soon as it is added to the toplevel.